skietbaan_fe/src/components/RegisterMemberStyles.css

```diff
@@ -32,15 +28,19 @@
     box-shadow: 2px 2px rgba(0,0,0,0.24);
   }
   @media screen and (min-width:768px)  {
-  .centre-register{
+  .centre-registermember{
     width: 50%;
     left:25%;
-    top:1%;
+    top:3%;
```

Comment:
There no better way to css? Add it in a container or something, or if you use boodstrap- use boodstrap if you use flexboxes-use flexboxes